src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -45,6 +47,7 @@ class BrandedCheckoutStep1Controller {
         break
     }
     this.itemConfig['recurring-day-of-month'] = this.day
+    this.itemConfig.frequency = this.frequency
```

Comment:
This seems unrelated but it must be needed for some reason? AngularJS makes the flow of data really hard to follow. Maybe something has always been broken?

Reply:
It seems when the initial implementation was done, just the bare minimum was brought into this branded checkout `itemConfig` object. It's not that something was broken, but that this value wasn't used, so it wasn't put in the object.